base/applications/network/tracert/tracert.cpp

```diff
@@ -577,15 +734,17 @@ ParseCmdline(int argc, wchar_t *argv[])
                 break;
 
             case 'h':
-                Info.MaxHops = GetULONG(argv[++i]);
+                if (GetOptionNumberOfHops(argc, argv, &i, &Info.MaxHops) == false)
```

Comment:
```suggestion
                if (!GetOptionNumberOfHops(argc, argv, &i, &Info.MaxHops))
```

Reply:
either is fine, this is just personal preference?